raiden-ts/src/path/epics.ts

```diff
@@ -164,3 +145,71 @@ export const pathFindServiceEpic = (
       );
     }),
   );
+
+/**
+ * Sends a PFSCapacityUpdate to PFS global room on new deposit on our side of channels
+ *
+ * @param action$ - Observable of channelDeposited actions
+ * @param state$ - Observable of RaidenStates
+ * @returns Observable of messageGlobalSend actions
+ */
+export const pfsCapacityUpdateEpic = (
+  action$: Observable<RaidenAction>,
+  state$: Observable<RaidenState>,
+  { address, network, signer, config$ }: RaidenEpicDeps,
+): Observable<ActionType<typeof messageGlobalSend>> =>
+  action$.pipe(
+    filter(isActionOf(channelDeposited)),
```

Comment:
More of a question: Shouldn't we inform the PFS on `channelWithdrawn` events about updated capacities, since these events alter the channel capacity as well? I think we don't fire these yet, but we already listen for them when they happen.

Reply:
But your coment made me realise we should consider the withdraw amount on capacity calculation, so I'll get that on as well

Reply:
Thanks for clarifying @andrevmatos